packages/hash/api/src/seed-data/dev-users.ts

```diff
@@ -10,18 +10,27 @@ type DevelopmentUser = {
   email: string;
   shortname: string;
   preferredName: string;
+  isInstanceAdmin: boolean;
```

Comment:
nit: i suppose this could be optional

Reply:
Yes it could, have done so in https://github.com/hashintel/hash/pull/1327/commits/aac906ec8a4c3965ff981662124732571aa601fb